support-frontend/assets/components/checkoutSupportOnce/checkoutSupportOnceButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { css } from '@emotion/react';
+import {
+	Button,
+	SvgChevronRightSingle,
+} from '@guardian/source-react-components';
+
+const button = css`
+	color: #606060;
+	background-color: transparent;
+	&:hover {
+		background-color: transparent;
+	}
+	& svg {
+		width: 24px;
+		margin: 2px;
+	}
+`;
+
+interface CheckoutSupportOnceButtonProps {
+	onClick: () => void;
+}
+
+export function CheckoutSupportOnceButton({
+	onClick,
+}: CheckoutSupportOnceButtonProps): JSX.Element {
+	return (
+		<Button
+			onClick={onClick}
+			icon={<SvgChevronRightSingle size="xsmall" />}
+			size="small"
+			hideLabel
```

Comment:
What label is being hidden here? A button should _always_ have some text content, even if it's not visible, for screen reader users.

Reply:
hideLabel is incorrect, it's also not required to be clickable if inside a clickable component. Have renamed to 'CheckoutSupportOnceChevronRight' component which contains just a div and chevron right styled. Possibly does not need to be a component?